pyproject.toml

```
@@ -47,6 +47,7 @@ gpib-ctypes = ["gpib-ctypes>=0.3.0"]
 serial = ["pyserial>=3.0"]
 usb = ["pyusb"]
 psutil = ["psutil"]
+zeroconf = ["zeroconf"]
```

```suggestion
hislip-discovery = ["zeroconf"]
```

This will make it easier to discover I believe (could be worth adding to the docs). I should also change the psutil one.